op-node/rollup/attributes/engine_consolidate_test.go

```diff
@@ -74,12 +78,17 @@ func holoceneArgs() matchArgs {
 			Transactions:          []eth.Data{validTxData},
 			Withdrawals:           &types.Withdrawals{},
 			SuggestedFeeRecipient: validFeeRecipient,
-			EIP1559Params:         validHoloceneEIP1559Params,
+			EIP1559Params:         validJovianEIP1559Params,
 		},
 		parentHash: validParentHash,
 	}
 }
 
+func holoceneArgs() matchArgs {
+	args := jovianArgs()
+	return args
+}
+
```

Comment:
At a glance, I think we want to keep the existing `holoceneArgs` and just update the type for the extra data. `jovianArgs` should be separate with a value set for minBaseFeeLog2.